OpenSim/Simulation/Wrap/WrapCylinder.cpp

```diff
@@ -125,7 +125,7 @@ void WrapCylinder::extendFinalizeFromProperties()
     {
         string errorMessage = "Error: radius for WrapCylinder " + getName() +
             " was either not specified, or is negative.";
-        throw Exception(errorMessage);
+        OPENSIM_THROW_FRMOBJ(InvalidPropertyValue, errorMessage);
```

Comment:
The `errorMessage` argument is passed as `InvalidPropertyValue`'s `propertName` parameter, so the resulting message will not be correct. I think `InvalidPropertyValue` should be updated to take a general message in addition to the property name (also, the `propertName` parameter has a typo in the name).

Lastly, I don't think the message passed to `InvalidPropertyValue` should include `Error: `. 

These comments apply to all similar changes in this PR.